app/controllers/groups_controller.rb

```diff
@@ -10,15 +10,24 @@ def index
 
   def create
     CreateGroupAssignments.new(user_ids, group_params).call
+    go_back_with_success
+  end
+
+  def destroy
+    DestroyGroupAssignments.new(user_ids, params[:id]).call
+
+    go_back_with_success(:removed)
+  end
 
+  private
+
+  def go_back_with_success(action = :assigned)
```

Comment:
I would make this parameter not optional so it's more obvious when you use it in `create`.